### ISSUE TITLE
Column name USER_COMMENT causes column parsing to end

### ISSUE THREAD

**Issue by stanorama** [xnuinside/simple-ddl-parser#128]
**Describe the bug**

There are 2 issues with parsing the enclosed create table statement in Snowflake.
1. The 'OR REPLACE' keywords cause the output to be blank.
2. The column with name 'USER_COMMENT' causes the column parsing to end so the output only includes the first column 'ASIN'.

Also a feature request: Support for constraint and table comment parsing

**To Reproduce**

```
create or replace TABLE ASIN.EXCLUSION (
	ASIN VARCHAR(10) NOT NULL,
	USER_COMMENT VARCHAR(100),
	PROCESS_SQN NUMBER(10,0) NOT NULL,
	constraint PK_EXCLUSION primary key (ASIN)
) COMMENT='ASINs to be excluded from the ASIN List File'
;

```

```
create TABLE ASIN.EXCLUSION (
	ASIN VARCHAR(10) NOT NULL,
	USER_COMMENT VARCHAR(100),
	PROCESS_SQN NUMBER(10,0) NOT NULL,
	constraint PK_EXCLUSION primary key (ASIN)
) COMMENT='ASINs to be excluded from the ASIN List File'
;
```

**Comment by xnuinside** [comment]
@stanorama thanks for reporting the issue, will work on it soon!

**Comment by xnuinside** [comment]
@stanorama  one more time thanks for raising the issue, you help me to find a great bug ) problem was with all strings, that started with 'USE' and 'GO'. So it was really meaning issue. I fixed in 0.26.2 release & added tests - https://github.com/xnuinside/simple-ddl-parser/blob/main/tests/test_snowflake.py#L196. Please use 0.26.2 - already released. If you will find anymore problems - feel free to open new issue, its really helpful
